ci-operator/step-registry/openstack-k8s-operators/gather/openstack-k8s-operators-gather-commands.sh

```diff
@@ -5,11 +5,10 @@ set +eu
 
 MUST_GATHER_TIMEOUT=${MUST_GATHER_TIMEOUT:-"5m"}
 
-# OCP must-gather
 mkdir -p ${ARTIFACT_DIR}/must-gather/
-oc --insecure-skip-tls-verify adm must-gather --timeout=$MUST_GATHER_TIMEOUT \
---dest-dir ${ARTIFACT_DIR}/must-gather > ${ARTIFACT_DIR}/must-gather/must-gather.log
 
-# OSP must-gather
-oc --insecure-skip-tls-verify adm must-gather --image=quay.io/openstack-k8s-operators/openstack-must-gather:latest --timeout=$MUST_GATHER_TIMEOUT \
---dest-dir ${ARTIFACT_DIR}/must-gather > ${ARTIFACT_DIR}/must-gather/openstack-must-gather.log
+# Run the must-gather command
+oc --insecure-skip-tls-verify adm must-gather --image-stream=openshift/must-gather \
+    --image=quay.io/openstack-k8s-operators/openstack-must-gather:latest \
+    --timeout=$MUST_GATHER_TIMEOUT \
+    --dest-dir ${ARTIFACT_DIR}/must-gather -- ADDITIONAL_NAMESPACES=kuttl,sushy-emulator gather &> ${ARTIFACT_DIR}/must-gather/openstack-must-gather.log
```

Comment:
So the artifacts we used to get (and that we cared about) via the "OCP must-gather" are now covered by the "OSP must-gather" and its `ADDITIONAL_NAMESPACES` option?

Reply:
no, `ADDITIONAL_NAMESPACES` is a feature we need to dynamically pass (e.g., from ci-framework) a list of namespaces we'd like to inspect (see https://github.com/openstack-k8s-operators/ci-framework/pull/843). The
osp must-gather is relevant and still there, I just changed the syntax to execute both with a single command instead of two commands. In my understanding passing `--image-stream=openshift/must-gather` should be able to trigger in background the `OCP` gathering [1].
What do you think about this approach?

[1] https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/46865/rehearse-46865-pull-ci-openstack-k8s-operators-manila-operator-main-manila-operator-build-deploy-kuttl/1736658414899040256/artifacts/manila-operator-build-deploy-kuttl/openstack-k8s-operators-gather/artifacts/must-gather/

Reply:
Cool.  That sounds good to me.